.github/workflows/package-dependencies-update.yml

```diff
@@ -0,0 +1,78 @@
+name: Update package dependencies versions
+
+# Set jobs to be configured and executed by schedule.
+on:
+  push:
+    branches:
+      - 'main'
+  schedule:
+    - cron: '0 8 * * Tue' # Actions will be executed at 8am for every Tuesday.
```

Comment:
Can we run every weekday? Goal is to be faster than the secure engineering checks. 😄 
Maybe run earlier when there's less activity.

Reply:
We can run every weekday, let me adjust that with the code. 👍🏽

Reply:
Also good for testing!